app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -39,12 +63,28 @@ export const ListingDetailsEligibility = ({
       )}
       <ListSection
         title={t("listings.householdMaximumIncome")}
-        subtitle={t("listings.forIncomeCalculations")}
+        subtitle={
```

Comment:
don't think this is valid html putting block level elements inside inline elements. should we change the span to a div?

also, why not use styling instead of br's?